developer.md

```diff
@@ -0,0 +1,34 @@
+# Python Client Library
+
+### Documentation
+
+   [stable](http://genestack-client.readthedocs.io/en/stable/)
+
+### Installation
+
+   - install library
+     - `pip install genestack-client --upgrade`
+   - setup default user
+     - `genestack-user-setup init`
+
+### Repo
+
+  ```
+  docs/                             - folder with documentation sources
+  genestack_client/                 - folder with library
+  tests/                            - developer tests, require pytest to run.
+  genestack-application-manager     - script to connect to application-manager (installed as script)
+  genestack-user-setup              - script to setup user (installed as script)
+  genestack-uploader                - commandline uploader (installed as script)
+  genestack-shell                   - shell (installed as script)
+  setup.py                          - installation setup for genestack_client library
+  ```
+
+### Generate documentation files
+
+ - install [**sphinx**](http://sphinx-doc.org/) and **sphinx-rtd-theme**
+   - `pip install Sphinx sphinx-rtd-theme`
+ - `cd docs`
+ - `make html`
+
+**Note!** `make` use script `make_shell_docs.py` that generates documentation for shell applications
```

Comment:
If you are set on creating separate files for whatever reason (I don't see any need for that whatsoever), call it `README.dev.md`.

Reply:
Do you think that current readme is ok for external developers? Even `Generate documentation files` part?

Reply:
Maybe, `readme` should stay the same, but PyPi description should be updated? Because current readme doesn't look suitable for PyPi description, too many internal details.

Reply:
Why not?  If we are talking about external _developers_ (not just _users_), then how can they improve docs (do "some gramma fixes", for example ;) without generating documentation locally?  current README is quite short and I see no problems for users/dev in skipping sections they aren't interested in.

Reply:
I agree with whole README being unsuitable for PyPI description.  Here's how it's done in _jinja_ project: https://github.com/pallets/jinja/blob/master/setup.py#L41 (and I like this approach).

Reply:
Good example. Ideally a description should contain a simple usage example, but I can't think of anything like that for this python client.

Reply:
I don't think it's suitable, so let's leave it for full documentation.